be/src/common/config.h

```diff
@@ -1020,9 +1020,9 @@ CONF_Int64(max_length_for_bitmap_function, "1000000");
 
 // Configuration items for datacache
 CONF_Bool(datacache_enable, "false");
-CONF_mString(datacache_mem_size, "10%");
-CONF_mString(datacache_disk_size, "0");
-CONF_mString(datacache_disk_path, "${STARROCKS_HOME}/datacache/");
+CONF_String(datacache_mem_size, "10%");
+CONF_String(datacache_disk_size, "0");
+CONF_String(datacache_disk_path, "${STARROCKS_HOME}/datacache/");
 CONF_String(datacache_meta_path, "${STARROCKS_HOME}/datacache/");
```

Comment:
If you change mString to String, all immutable string configure can be changed too.
But I don't think this is safe for all configurations.

Reply:
I changed thse three configurations to immutable configurations becauses they are actually immutable: modifying their values at runtime has no real effect.

Reply:
I've already confirmed this with @GavinMar

Reply:
> ... But I don't think this is safe for all configurations.

Could you please give a more detailed description, I don't quite understand what you mean.